openfl/component/collaborator/collaborator.py

```diff
@@ -68,18 +77,18 @@ class Collaborator:
     """
 
     def __init__(self,
-                 collaborator_name,
-                 aggregator_uuid,
-                 federation_uuid,
-                 client,
-                 task_runner,
-                 task_config,
-                 opt_treatment='RESET',
-                 device_assignment_policy='CPU_ONLY',
-                 delta_updates=False,
-                 compression_pipeline=None,
-                 db_store_rounds=1,
-                 **kwargs):
+                 collaborator_name: str,
+                 aggregator_uuid: str,
+                 federation_uuid: str,
+                 client: Any,
+                 task_runner: Any,
```

Comment:
Now we use `CoreTaskRunner` I think, so:
```python3
from openfl.federated.task.task_runner import CoreTaskRunner
```
```suggestion
                 task_runner: CoreTaskRunner,
```